packages/compass-aggregations/src/components/saved-pipelines/saved-pipeline-confirmation-modals.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { ConfirmationModal } from '@mongodb-js/compass-components';
+
+export const OpenPipelineConfirmationModal: React.FunctionComponent<{
+  isOpen: boolean;
+  onCancel: () => void;
+  onOpen: () => void;
```

Comment:
I think the naming here has some potential for confusion because the word “open” has a different meaning in “isOpen” vs. “onOpen”; maybe we should just use `onConfirm` for the props of this component as well?